mods/ts/rules/gdi-vehicles.yaml

```
@@ -425,3 +425,35 @@ JUGG:
 		ArmamentNames: deployed
 	Selectable:
 		DecorationBounds: 1448, 2413, 0, -482
+
+MOBILEMP:
+	Inherits: ^Tank
+	Inherits@VOXELS: ^VoxelActor
+	Inherits@selection: ^SelectableSupportUnit
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 130
+		Prerequisites: ~factory, napuls, ~techlevel.superweapons
```

```suggestion
		Prerequisites: ~gaweap, napuls, ~techlevel.superweapons
```

as it is otherwise visible for Nod but can never be built.